src/core_plugins/kibana/public/dashboard/components/full_screen_placeholder.js

```diff
@@ -0,0 +1,56 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import chrome from 'ui/chrome';
+
+import {
+  KuiKeyboardAccessible,
+} from 'ui_framework/components';
+import {
+  keyCodes,
+} from 'ui_framework/services';
+
+export class FullScreenModePlaceholder extends PureComponent {
```

Comment:
What is the word "placeholder" intended to mean in this case? I'm trying to find a place for it in my mental map of our UI.

Reply:
Maybe just calling it `ExitFullScreenButton` would be clearer?